src/components/SkillsSection.vue

```diff
@@ -30,4 +30,7 @@ export default {
 </template>
 
 <style scoped>
+.skills {
+    scroll-margin-top: 60px;

```

Comment:
Create new style rule and apply it for every section. 
This rule have to contain this style.